### PR TITLE
[FIX] l10n_cl_etd_stock: Expected singleton

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

File: l10n_cl_etd_stock/models/stock_picking.py [hunk 1]
```diff
@@ -15,11 +15,11 @@ def _compute_class_id_domain(self):
 
     @api.multi
     def action_done(self):
-        res = super().action_done()
-        sign = self._name in [x.model for x in self.company_id.etd_ids]
-        for picking in self:
-            if sign and picking.location_dest_id.usage == 'customer':
-                self.with_delay().document_sign()
+        for rec in self:
+            res = super(StockPicking, rec).action_done()
```
Comment (bodedra) [inline l10n_cl_etd_stock/models/stock_picking.py]:
```suggestion

```

File: l10n_cl_etd_stock/models/stock_picking.py [hunk 2]
```diff
@@ -15,11 +15,11 @@ def _compute_class_id_domain(self):
 
     @api.multi
     def action_done(self):
-        res = super().action_done()
-        sign = self._name in [x.model for x in self.company_id.etd_ids]
-        for picking in self:
-            if sign and picking.location_dest_id.usage == 'customer':
-                self.with_delay().document_sign()
+        for rec in self:
```
Comment (bodedra) [inline l10n_cl_etd_stock/models/stock_picking.py:19]:
```suggestion
        res = super(StockPicking, self).action_done()
        for rec in self:
```

**Review by bodedra — APPROVED** [review]
Added minor suggestion.

**Review by nelsonramirezs — APPROVED** [review]
LGTM

**Review by nelsonramirezs — APPROVED** [review]

**Pull request merged**